packages/shared/src/react/hooks/useReverification.ts

```diff
@@ -14,7 +14,7 @@ const CLERK_API_REVERIFICATION_ERROR_CODE = 'session_reverification_required';
 async function resolveResult<T>(result: Promise<T> | T): Promise<T | ReturnType<typeof reverificationError>> {
   try {
     const r = await result;
-    if (r instanceof Response) {
+    if (typeof Response !== 'undefined' && r instanceof Response) {
```

Comment:
❓ Just curious, did this ever run in an env where Response was not defined as a global identifier?